frontend/packages/client/package.json

```diff
@@ -43,7 +47,7 @@
     "react-hook-form": "^7.11.0",
     "react-images-upload": "^1.2.8",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "5.0.1",
```

Comment:
upgrade to react scripts 5 because of this bug: https://github.com/framer/motion/issues/1525

also semi-related I sometimes get this bug with v4 that is mad annoying https://stackoverflow.com/questions/69051008/react-injecting-iframe-with-max-z-index-on-reload-after-changes-development